redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -46,7 +48,7 @@ DatabaseServerV4Responses list(@QueryParam("environmentId") String environmentId
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = DatabaseServerOpDescription.GET_BY_NAME, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
         nickname = "getDatabaseServer")
-    DatabaseServerV4Response get(@PathParam("name") String name);
+    DatabaseServerV4Response get(@PathParam("environmentId") String environmentId, @PathParam("name") String name);
```

Comment:
This PathParam won't work - there needs to be "{environmentId}" somewhere in the value for the Path annotation. Now - if redbeams actually still starts with this in place, then I'd be OK with it and we can fix the path specifications later, depending on whether it can be a path or query parameter. But if redbeams doesn't start, I'd call this a no-go.

Same for the others.

Reply:
Oof, for some reason I missed that these were @PathParams and thought they were get params -- my bad, will fix.